source/pdo_sqlsrv/pdo_parser.cpp

```diff
@@ -417,7 +417,9 @@ void sql_string_parser::parse_sql_string( TSRMLS_D ) {
     try {
         while ( !this->is_eos() ) {
             int start_pos = -1;
-
+            if ( pos == -1 ) {
+                next();
```

Comment:
In the constructor of the sql_string_parser (https://github.com/Microsoft/msphpsql/blob/dev/source/pdo_sqlsrv/pdo_parser.cpp#L41), and pos is initialized as -1. I didn't want to initialize it to 0 since conn_string_parser has it initilized as -1